source/braille.py

```diff
@@ -1659,6 +1659,8 @@ def nextLine(self):
 				return
 		dest.collapse()
 		self._setCursor(dest)
+		dest.expand(self._getReadingUnit())
```

Comment:
I'm afraid this can become problematic, since you're mutating the TextInfo that was said as cursor before. Instead, I think you should copy the textInfo first.
```suggestion
		copy = dest.copy()
		copy.expand(self._getReadingUnit())
```